docs/package.json

```diff
@@ -3,12 +3,16 @@
   "version": "0.1.0",
   "scripts": {
     "build": "webpack",
+    "build:clean": "rimraf build",
     "start": "webpack-dev-server --mode development --content-base docs/build/ --hot --progress --colors --host 0.0.0.0",
     "start:stats": "webpack --display-modules --display-chunks --json > stats.json --progress"
   },
   "dependencies": {
     "@babel/register": "^7.4.4",
     "core-js": "^3.1.3",
+    "lodash": "^4.17.14",
+    "prop-types": "^15.7.2",
+    "react": "^16.8.6",
```

Comment:
NAB: A little curious to know if there is any issues that come along with adding react here. Like, when running the docs build if there is any errors yelling at us about having two react modules installed.

Reply:
@morethanfire I hadn't noticed any issues with adding it.  And ESLINT was complaining to me about not having it.  I assume since merging this things are all still happy.

@morethanfire @IlyaRadinsky I guess I can try and handle the `onTouch*` events for mobile/touch support in a subsequent PR?  Is the browser based emulator typically good enough for testing this, or is NGrok and actual device testing recommended?